CHANGELOG.md

```diff
@@ -6,6 +6,26 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [PEP 440](https://www.python.org/dev/peps/pep-0440/) 
 and uses [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.15.0]
+
+### Removed
+- The [`reusable-mypy`](.github/workflows/reusable-mypy.yml) workflow no longer passes any optional arguments to the `mypy` command. This makes it easier to run `mypy` locally, because the command is just:
+
+  ```
+  mypy .
+  ```
+
+  If you want to preserve the removed options for your project, you must now add them to `[tool.mypy]` in `pyproject.toml`:
```

Comment:
I think this excerpt (through line 27) would be better to spell out in the README file. Unless it works better here?

Reply:
This is what I would want to to be included in the release notes for whoever merges the dependabot PRs for this release (probably me), so that they can just copy-paste those options into `pyproject.toml` for each repo when they merge the dependabot PR.

Reply:
I also added the options (other than `ignore_missing_imports) to the README so that we'll use them for any new projects.

Reply:
Line 27 "Note that we discourage enabling `ignore_missing_imports` [...]" I figured only applies to this release, since the `ignore_missing_imports` option isn't mentioned in the README.

Reply:
Sounds good! Thanks!